src/stateless/selectbox.vue

```diff
@@ -69,6 +71,8 @@ export default {
             isOpen: false,
             focused: false,
             searchTextValue: '',
+            isSearchLoading: false,
```

Comment:
Do we need this in data? Couldn't we store a promise and check if it is resolved?

Reply:
Can't vue reactively handle promise state?

Reply:
We could probably have a computed then